Add DEA Intertidal product page (#173)
GeoscienceAustralia/dea-knowledge-hub: docs/data/product/dea-intertidal/_access.md

% ## Access constraints

% ## Use constraints

:::{dropdown} How to access the data

To download DEA Intertidal data:

> 1. Click the [Link to data](https://data.dea.ga.gov.au/?prefix=derivative/ga_s2ls_intertidal_cyear_3/) link above

Reviewer: ```suggestion
1. Click the [Access the data on AWS](https://data.dea.ga.gov.au/?prefix=derivative/ga_s2ls_intertidal_cyear_3/) link above.
```

Reviewer: @stevesagar Minor wording suggestion

% ## Access constraints

% ## Use constraints

:::{dropdown} How to access the data

To download DEA Intertidal data:
1. Click the [Link to data](https://data.dea.ga.gov.au/?prefix=derivative/ga_s2ls_intertidal_cyear_3/) link above

> 2. Click on the GRID file to download to your computer. This file can be used in a GIS package to identify the product tiles that you require for a given location. Alternatively, you can access this file via DEA maps (LINK) to identify required tiles. 

Reviewer 2: ```suggestion
2. Click on the [ga_summary_grid_c3_32km_coastal.geojson](https://data.dea.ga.gov.au/?prefix=derivative/ga_s2ls_intertidal_cyear_3/ga_summary_grid_c3_32km_coastal.geojson) file to download to your computer. This file can be used in a GIS package to identify the product tiles that you require for a given location. Alternatively, you can access this file via DEA Maps to identify required tiles (`Sea, ocean and coast > DEA Intertidal > DEA Intertidal 32 km tile grid`). 
```

3. Navigate to the required tile folder, first using the tile x reference (e.g. `x079`) and then the tile y reference (e.g. `y123`). Then select your year of interest.
4. Click on the required product layer to download. See [Technical Information](./?tab=details#technical-information) for details on file naming and product layer details.  

:::

:::{dropdown} How to explore DEA Maps
To explore DEA Intertidal on the interactive DEA Maps platform, visit the link below:

> [https://maps.dea.ga.gov.au/story/DEAIntertidal](https://maps.dea.ga.gov.au/#share=s-DEAIntertidal&playStory=1)

Reviewer 2: ```suggestion
[https://maps.dea.ga.gov.au/story/DEAIntertidal](https://maps.dea.ga.gov.au/story/DEAIntertidal)
```
